dash-renderer/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": [
+      "@plotly/eslint-config-dash/typescript",
+    ],
+    "rules": {
+      "no-constant-condition": 0,
+      "no-prototype-builtins": 0,
+      "no-unused-vars": 0,
```

Comment:
The renderer's configuration has some peculiarities that shouldn't be generalized to all projects. Also `no-unused-vars` just acknowledges the current state of the JavaScript code in this project not adhering to this rule.. there's a lot of dangling unused variables in JS files at the moment 😢

Reply:
Hmm OK, would be nice to tighten up `no-unused-vars` at some point, and see whether the others actually have a legitimate reason to be violated. Care to make an issue to follow up on this later?

Reply:
I see, I missed a detail here: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md
```
// note you must disable the base rule as it can report incorrect errors
  "no-unused-vars": "off",
```

Also used the occasion to fix the `.eslintrc` config so that it is loaded correctly by VSCode when it referenced a tsconfig.

https://github.com/plotly/dash/pull/1578/commits/2fef7ed75398706233282d98cdaadb975af425b3